components/script/dom/console.rs

```diff
@@ -65,13 +73,76 @@ fn console_message(global: &GlobalScope, message: DOMString, level: LogLevel) {
     })
 }
 
+///Generates matrix of table elements, and then converts matrix into a properly formatted DOMString
```

Comment:
When making a comment in rust you should put a space after the slashes. In this case, I don't think the rustdoc will work without the space.

Reply:
Oh ok sure! Thanks for the suggestion, I'm not really familiar with rustdoc so that really helps. I'll make sure to fix the comment.